### PR TITLE
Update run-example-test-pip.sh for image-augmentation and  image-augmentation-3d

### PR DESCRIPTION
**Pull request by insularZJ**
add test for image-augmentation

### CODE REVIEW

**Comment by insularZJ** [comment]
@hkvision

File: apps/run-app-tests-pip.sh [hunk 1]
```diff
@@ -11,6 +11,54 @@ clear_up () {
...
+
+# Run the example
+export SPARK_DRIVER_MEMORY=1g
+python ${ANALYTICS_ZOO_HOME}/apps/image-augmentation/image-augmentation.py
+
+exit_status=$?
+if [ $exit_status -ne 0 ];
+then
+    clear_up
+    echo "anomaly-detection failed"
```
Comment (hkvision) [inline apps/run-app-tests-pip.sh]: wrong name

File: apps/run-app-tests-pip.sh [hunk 2]
```diff
@@ -11,6 +11,54 @@ clear_up () {
...
+    exit $exit_status
+fi
+
+unset SPARK_DRIVER_MEMORY
+now=$(date "+%s")
+time9=$((now-start))
+echo "#9 image-augmentation time used:$time9 seconds"
+
+echo "#11 start app test for image-augementation-3d"
```
Comment (hkvision) [inline apps/run-app-tests-pip.sh]: typo

File: apps/run-app-tests-pip.sh [hunk 3]
```diff
@@ -11,6 +11,54 @@ clear_up () {
...
+
+# Run the example
+export SPARK_DRIVER_MEMORY=1g
+python ${ANALYTICS_ZOO_HOME}/apps/image-augmentation-3d/image-augementation-3d.py
+
+exit_status=$?
+if [ $exit_status -ne 0 ];
+then
+    clear_up
+    echo "anomaly-detection failed"
```
Comment (hkvision) [inline apps/run-app-tests-pip.sh]: typo

File: apps/run-app-tests-pip.sh [hunk 4]
```diff
@@ -313,3 +361,5 @@ echo "#12 image_classification_inference time used:$time12 seconds"
 
 # This should be done at the very end after all tests finish.
 clear_up
+
```
Comment (hkvision) [inline apps/run-app-tests-pip.sh]: remove redundant lines

**Comment by hkvision** [comment]
LGTM. Can merge after jenkins pass

**Comment by hkvision** [comment]
jenkins passed http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Pip-Apps-Integration/135/

**Pull request merged**